.pre-commit-config.yaml

```diff
@@ -21,4 +21,4 @@ repos:
     rev: 22.10.0
     hooks:
       - id: black
-        language_version: python3.9
+        language_version: python3
```

Comment:
we shouldn't change this, the more specific we can get the better- if there is a reason to change it, let me know